### PR TITLE
docs: spanish translation

### LINKED ISSUE #776: docs(i18n): add spanish translation
### Is your feature request related to a problem? Please describe.

n/a

### Describe the solution you'd like to see

Add spanish translations of the brand new [create.t3.gg](https://create.t3.gg) documentation

### Describe alternate solutions

n/a

### Additional information

Submitting this issue to spread awareness and avoid multiple translation versions.

I'll probably assign myself this issue on the weekend, but if any other contributor already started this, I'll be happy to review the PR.

### PR DESCRIPTION
**Pull request by andrescv**
Closes #776

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Spanish translation

- [x] deployment
    - [x] docker.md
    - [x] netlify.md
    - [x] vercel.md
- [x] usage
    - [x] env-variables.md
    - [x] first-steps.md
    - [x] next-auth.md
    - [x] next-js.md
    - [x] prisma.md
    - [x] tailwind.md
    - [x] trpc.md
    - [x] typescript.md
- [x] faq.md
- [x] folder-structure.mdx
- [x] installation.md
- [x] introduction.md
- [x] other-specs.md
- [x] t3-collection.mdx
- [x] why.md

---
Screenshot 2022-12-04 at 5 35 30 PM

💯

### CODE REVIEW

File: www/src/pages/es/deployment/docker.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+USER nextjs
+EXPOSE 3000
+ENV PORT 3000
+
+CMD ["node", "server.js"]
+
+```
+
+> **_Notes_**
```
Comment (hodeinavarro) [inline www/src/pages/es/deployment/docker.md:128]: Debería ser `Notas`

Comment (hodeinavarro) [inline www/src/pages/es/deployment/docker.md:128]: Debería ser Notas

File: www/src/pages/es/faq.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+layout: ../../layouts/docs.astro
+---
+
+Aquí hay algunas preguntas frecuentes sobre `create-t3-app`.
+
+## ¿Qué sigue? ¿Cómo hago una aplicación con esto?
+
+Tratamos de mantener este proyecto lo más simple posible, para que pueda comenzar solo con el esqueleto que configuramos para usted y agregar cosas adicionales más adelante cuando sea necesario.
+
+Si no está familiarizado con las diferentes tecnologías utilizadas en este proyecto, consulte los documentos respectivos. Si todavía no está seguro, únase a nuestro [Discord](https://t3.gg/discord) y solicite ayuda.
```
Comment (hodeinavarro) [inline www/src/pages/es/faq.md:13]: Yo diría la "documentación" respectiva.

File: www/src/pages/es/faq.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## ¿Qué recursos de aprendizaje están disponibles actualmente?
+
+Aunque los recursos que se enumeran a continuación son algunos de los mejores que existen para T3 Stack, la comunidad (y [Theo](https://youtu.be/rzwaaWH0ksk?t=1436)) recomiendan que comience a usar el stack y aprenda en el camino construyendo con él.
+
+Si está considerando `create-t3-app`, es probable que ya haya usado algunas de las partes del stack. Entonces, ¿por qué no simplemente sumergirse en el y aprender las otras partes mientras construye algo?
+
+Ahora, sabemos que este camino no funciona para todos. Por lo tanto, si cree que ha probado las recomendaciones y todavía le gustarían algunos recursos, o simplemente no está seguro de hacerlo solo y/o se siente abrumado por el stack, consulte estos increíbles tutoriales en `create-t3-app `:
+
+### Artículos
```
Comment (hodeinavarro) [inline www/src/pages/es/faq.md:29]: Habrá que estar atentos a cómo traduce la comunidad los títulos de los artículos, ya que es posible que no sean traducidos y no llevemos una línea de continuidad entre los diferentes idiomas.

File: www/src/pages/es/faq.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+- [Creando una aplicación full stack con create-t3-app](https://www.nexxel.dev/blog/ct3a-guestbook)
+- [Un primer vistazo a create-t3-app](https://dev.to/ajcwebdev/a-first-look-at-create-t3-app-1i8f)
+- [Migrando su aplicación T3 a Turborepo](https://www.jumr.dev/blog/t3-turbo)
+
+### Vídeos
```
Comment (hodeinavarro) [inline www/src/pages/es/faq.md:35]: Idem.

File: www/src/pages/es/introduction.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+<div class="embed">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/PbjHxIuHduU" title="El mejor stack para su próximo proyecto" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+## El T3 Stack
+
+El _"T3 Stack"_ es un stack de desarrollo web creado por [Theo](https://twitter.com/t3dotgg) que está enfocado en simplicidad, modularidad, y seguridad (full-stack) de tipos.
+
+Las piezas centrales son [**Next.js**](https://nextjs.org/) y [**TypeScript**](https://typescriptlang.org/). [**Tailwind CSS**](https://tailwindcss.com/) está casi siempre incluído.
+Si está haciendo algo parecido a un backend, [**tRPC**](https://trpc.io/), [**Prisma**](https://prisma.io/), y [**NextAuth.js**](https://next-auth.js.org/) son muy buenas adiciones también.
```
Comment (hodeinavarro) [inline www/src/pages/es/introduction.md:16]: El salto de línea se debe a algo estético al visualizar en la web?

File: www/src/pages/es/introduction.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+Es posible que haya notado que hay muchas... piezas, esto es por diseño. Intercambiar piezas dentro y fuera según lo necesite. Este stack es modular desde las bases :)
```
Comment (hodeinavarro) [inline www/src/pages/es/introduction.md:18]:
```diff
-Intercambiar piezas dentro y fuera según lo necesite.
+Intercambie piezas dentro y fuera según lo necesite.
```

Comment (hodeinavarro) [inline www/src/pages/es/introduction.md:18]:
```diff
-Este stack es modular desde las bases :)
+Este stack es modular desde la base :)
```

File: www/src/pages/es/introduction.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+## Entonces... ¿Qué es create-t3-app? ¿Una plantilla?
+
+¿Más o menos? `create-t3-app` es una CLI creada por desarrolladores experimentados del T3 Stack para agilizar la configuración de una aplicación modular T3 Stack. Esto significa que cada pieza es opcional, y la "plantilla" es generada en función de sus necesidades específicas.
+
+Después de innumerables proyectos y muchos años en esta tecnología, tenemos muchas opiniones e ideas. Hemos hecho todo lo posible para codificarlos en esta CLI.
+
+Esta **NO** es una plantilla con todo incluido. **Esperamos** que traiga sus propias librerias que resuelvan las necesidades de **SU** aplicación. Si bien no queremos prescribir soluciones a problemas más específicos, como la administración de estado o la implementación de la aplicación como tal, [tenemos algunas recomendaciones enumeradas aquí](/es/other-recs).
```
Comment (hodeinavarro) [inline www/src/pages/es/introduction.md:26]:
```diff
-libreria
+librería
```

File: www/src/pages/es/introduction.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## Axiomas T3
+
+Seremos francos: este es un _proyecto de opinión_. Compartimos un puñado de creencias fundamentales sobre la construcción y las tratamos como la base de nuestras decisiones.
```
Comment (hodeinavarro) [inline www/src/pages/es/introduction.md:30]:
```diff
+Seremos francos: este es un _proyecto de opinión_.
-Seremos francos: este es un _proyecto dogmático_.
```

Comment (hodeinavarro) [inline www/src/pages/es/introduction.md:30]: >Compartimos un puñado de creencias fundamentales sobre la construcción y las tratamos como la base de nuestras decisiones.

Aunque `puñado` sea la traducción correcta, no estoy seguro de si es la más ideal... ¿Tal vez quedaría mejor `Compartimos varias creencias` o `Compartimos muchas creencias`?

File: www/src/pages/es/introduction.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+### Resolver Problemas
+
+Es fácil caer en la trampa de "agregar todo"; explícitamente no queremos hacer eso. Todo lo agregado a `create-t3-app` debería resolver un problema específico que existe dentro de las tecnologías principales incluidas. Esto significa que no agregaremos cosas como librerias de manejo de estado (`zustand`, `redux`), pero agregaremos cosas como NextAuth.js e integraremos Prisma y tRPC para usted.
+
+### Responsablemente Vanguardista
+
+Nos encanta nuestra tecnología de vanguardia. La velocidad y, sinceramente, la diversión que se obtiene de las nuevas tecnologías es realmente genial. Creemos que es importante ser vanguardista de manera responsable, usando tecnología más riesgosa en las partes menos riesgosas. Esto significa que no apostaríamos ⛔️ por una nueva tecnología de base de datos arriesgada (¡SQL es genial!). Pero felizmente ✅ apostamos por tRPC ya que son solo funciones que son triviales de remover.
```
Comment (hodeinavarro) [inline www/src/pages/es/introduction.md:38]:
```diff
-Creemos que es importante ser vanguardista de manera responsable, usando tecnología más riesgosa en las partes menos riesgosas.
+Creemos que es importante ser vanguardista de manera responsable, usando tecnología más riesgosa en las partes con menos riesgo.
```

**Review by hodeinavarro — CHANGES_REQUESTED** [review]

**Comment by gabrielelpidio** [comment]
Great work! @andrescv I think we should use "segunda persona (tu)" instead of "tercera persona(usted)" to refer to the reader, to make it less formal. Let me know your thoughts.

**Comment by andrescv** [comment]
@gabrielelpidio Thanks! and agree, I will do that.

File: www/src/pages/es/faq.md (hunk 3 above)
Comment (andrescv) [reply www/src/pages/es/faq.md:29]: Si, es algo a considerar. También la mayoría de estos artículos no están traducidos.

File: www/src/pages/es/introduction.md (hunk 5 above)
Comment (andrescv) [reply www/src/pages/es/introduction.md:16]: Me imagino que sí, así esta la version en inglés también.

**Inglés**
Screenshot 2022-11-19 at 8 49 57 AM

**Español**
Screenshot 2022-11-19 at 8 50 27 AM

File: www/src/pages/es/introduction.md (hunk 8 above)
Comment (andrescv) [reply www/src/pages/es/introduction.md]: Utilice `Compartimos muchas creencias`.

**Comment by juliusmarminge** [comment]
Please check out the new [TRANSLATIONS.md](https://github.com/t3-oss/create-t3-app/blob/next/www/TRANSLATIONS.md) and the notes in [config.ts](https://github.com/t3-oss/create-t3-app/blob/next/www/src/config.ts) and configure the sidebar for your language.

**Comment by Nsttt** [comment]
Hey, give me a ping once you have all of them so I can also take a look at everything :)

**Comment by andrescv** [comment]
If you guys @hodeinavarro @Nsttt can help me with a final review, I'd be really grateful 🙏🏼

**Comment by c-ehrlich** [comment]
please add this https://github.com/t3-oss/create-t3-app/pull/970

File: www/src/config.ts [hunk 10]
```diff
@@ -139,6 +165,11 @@ export const SIDEBAR_HEADER_MAP: Record<
   Exclude<KnownLanguageCode, "en">,
   Record<OuterHeaders, string>
 > = {
+  es: {
+    "Create T3 App": "Create T3 App",
```
Comment (amariani) [inline www/src/config.ts:169]:
```suggestion
    "Create T3 App": "Crear T3 App",
```

Comment (ramirofarias) [reply www/src/config.ts:169]: I wouldn't change this. "Create T3 App" is a name so it shouldn't be translated.

File: www/src/pages/es/usage/next-js.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+Utilizar Next.js junto con [Vercel](https://vercel.com/) hace que el desarrollo y despliegue de aplicaciones web sea más fácil que nunca. Su opción gratuita extremadamente generosa y su interfaz super intuitiva proporciona una solución de un solo clic para desplegar tu sitio (nosotros ❤️ Vercel)
+
+## Get Static/Server Props
+
+Una característica clave de Next.js son sus capacidades de obtención de datos. Recomendamos fuertemente leer la [documentación oficial](https://nextjs.org/docs/basic-features/data-fetching) para comprender cómo usar cada método y cómo difieren. `GetServersideProps` generalmente se desaconseja a menos que haya una buena razón para ello, debido al hecho de que es una llamada de bloqueo y reducirá el rendimiento de tu sitio. La [regeneración estática incremental](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) es una gran alternativa a `getServerSideProps` cuando los datos son dinámicos y se pueden obtener incrementalmente.
```
Comment (ramirofarias) [inline www/src/pages/es/usage/next-js.md:24]:
```suggestion
Una característica clave de Next.js son sus capacidades de obtención de datos. Recomendamos fuertemente leer la [documentación oficial](https://nextjs.org/docs/basic-features/data-fetching) para comprender cómo usar cada método y cómo difieren. `getServerSideProps` generalmente se desaconseja a menos que haya una buena razón para ello, debido al hecho de que es una llamada bloqueante y reducirá el rendimiento de tu sitio. La [regeneración estática incremental](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) es una gran alternativa a `getServerSideProps` cuando los datos son dinámicos y se pueden obtener incrementalmente.
```

File: www/src/pages/es/usage/prisma.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
+---
+title: Prisma
+description: Uso de Prisma
+layout: ../../../layouts/docs.astro
+lang: es
+---
+
+Prisma es un ORM para TypeScript, que te permite definir el esquema de tu base de datos y los modelos en un archivo `schema.prisma`, y luego generar un cliente seguro en tipos que se puede usar para interactuar con tu base de datos desde tu backend.
+
+## Cliente Prisma
```
Comment (ramirofarias) [inline www/src/pages/es/usage/prisma.md:10]:
```suggestion
## Prisma Client
```

File: www/src/pages/es/usage/prisma.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+
+Ubicado en `/server/db/client.ts`, el cliente prisma se instancia como una variable global (como se recomienda como [mejor práctica](https://www.prisma.io/docs/guides/database/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices#problem) por el equipo de Prisma) y exportado para ser utilizado en sus rutas API. Incluimos el cliente Prisma en el [contexto](/es/usage/trpc#-servertrpccontextts) de tRPC de forma predeterminada y recomendamos usar esto en lugar de importarlo por separado en cada archivo.
```
Comment (ramirofarias) [inline www/src/pages/es/usage/prisma.md:12]:
```suggestion
Ubicado en `/server/db/client.ts`, el Prisma Client se instancia como una variable global (como se recomienda como [mejor práctica](https://www.prisma.io/docs/guides/database/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices#problem) por el equipo de Prisma) y exportado para ser utilizado en tus rutas API. Incluimos el Prisma Client en el [contexto](/es/usage/trpc#-servertrpccontextts) de tRPC de forma predeterminada y recomendamos usar esto en lugar de importarlo por separado en cada archivo.
```

File: www/src/pages/es/usage/prisma.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+
+## Esquema
+
+Encontrarás el archivo de esquema de prisma en `/prisma/schema.prisma`. Este archivo es donde defines el esquema y los modelos de tu base de datos, y se usa al generar el cliente Prisma.
```
Comment (ramirofarias) [inline www/src/pages/es/usage/prisma.md:16]:
```suggestion
Encontrarás el archivo de esquema de Prisma en `/prisma/schema.prisma`. Este archivo es donde defines el esquema y los modelos de tu base de datos, y se usa al generar el cliente Prisma.
```

File: www/src/pages/es/usage/prisma.md [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+
+### Con NextAuth.js
+
+Cuando seleccionas NextAuth.js en combinación con prisma, el archivo de esquema se genera y se configura para ti con los valores recomendados para los modelos `User`, `Session`, `Account` y `VerificationToken`, según la [documentación de NextAuth.js](https://next-auth.js.org/adapters/prisma).
```
Comment (ramirofarias) [inline www/src/pages/es/usage/prisma.md:20]:
```suggestion
Cuando seleccionas NextAuth.js en combinación con Prisma, el archivo de esquema se genera y se configura para ti con los valores recomendados para los modelos `User`, `Session`, `Account` y `VerificationToken`, según la [documentación de NextAuth.js](https://next-auth.js.org/adapters/prisma).
```

File: www/src/pages/es/usage/tailwind.md [hunk 16]
```diff
@@ -0,0 +1,96 @@
+---
+title: Tailwind CSS
+description: Uso de Tailwind CSS
+layout: ../../../layouts/docs.astro
+lang: es
+---
+
+## ¿Qué es Tailwind CSS?
+
+Tailwind CSS es un pequeño framework, con [utilidad de primero](https://tailwindcss.com/docs/utility-first), de CSS para construir diseños personalizados, sin el cambio de contexto que requiere el CSS normal. Es puramente un framework de CSS y no proporciona componentes o lógica preconstruida, y proporciona [un conjunto muy diferente de beneficios](https://www.youtube.com/watch?v=CQuTF-bkOgc) en comparación con una librería de componentes como Material UI.
```
Comment (ramirofarias) [inline www/src/pages/es/usage/tailwind.md:10]:
```suggestion
Tailwind CSS es un pequeño framework [_utility-first_](https://tailwindcss.com/docs/utility-first) de CSS para construir diseños personalizados, sin el cambio de contexto que requiere el CSS normal. Es puramente un framework de CSS y no proporciona componentes o lógica preconstruida, y proporciona [un conjunto muy diferente de beneficios](https://www.youtube.com/watch?v=CQuTF-bkOgc) en comparación con una librería de componentes como Material UI.
```

File: www/src/pages/es/usage/tailwind.md [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+- [Integración de JetBrains](https://www.jetbrains.com/help/webstorm/tailwind-css.html#ws_css_tailwind_install)
+- [Neovim LSP](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#tailwindcss)
+
+### Formateo
+
+Las clases CSS de Tailwind pueden ser fácilmente desordenadas, por lo que es imprescindible un formateo para las clases. [El complemento de Prettier de Tailwind CSS](https://github.com/tailwindlabs/prettier-plugin-tailwindcss) clasifica las clases en el [orden recomendado](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier#how-classes-are-sorted) para que las clases coincidan con el paquete (_bundle_) CSS generado. Al seleccionar Tailwind CSS en la CLI, instalaremos y configuraremos esto para ti.
+
+### Aplicando clases condicionalmente
+
+Agregar condicionalmente clases con operadores terciarios puede ser muy desordenado y difícil de leer. Estos paquetes ayudan a organizar tus clases cuando se usa lógica condicional.
```
Comment (ramirofarias) [inline www/src/pages/es/usage/tailwind.md:81]:
```suggestion
Agregar condicionalmente clases con operadores ternarios puede ser muy desordenado y difícil de leer. Estos paquetes ayudan a organizar tus clases cuando se usa lógica condicional.
```

File: www/src/pages/es/why.md [hunk 18]
```diff
@@ -0,0 +1,50 @@
+---
+title: ¿Por qué CT3A?
+description: ¿Por qué deberías elegir Create T3 App para tu siguiente proyecto?
+layout: ../../layouts/docs.astro
+lang: es
+---
+
+Empezamos create-t3-app porque [Theo](https://twitter.com/t3dotgg) se negó a hacer una plantilla de sus tecnologías favoritas. Inspirándose en create-next-app, [CLI de Astro](https://astro.build) y un amor general por la seguridad de tipos, el equipo de create-t3-app trabajó arduamente para crear el mejor punto de partida posible para los nuevos proyectos de T3 Stack.
+
+Si está interesado en usar Next.js de forma segura, este es el lugar para comenzar. Si tienes curiosidad acerca de alguna de las opciones tecnológicas específicas que tomamos, sigue leyendo :)
```
Comment (ramirofarias) [inline www/src/pages/es/why.md:10]:
```suggestion
Si estás interesado en usar Next.js de forma segura, este es el lugar para comenzar. Si tienes curiosidad acerca de alguna de las opciones tecnológicas específicas que tomamos, sigue leyendo :)
```

File: www/src/pages/es/why.md [hunk 19]
```diff
@@ -0,0 +1,50 @@
...
+
+## ¿Por qué TypeScript?
+
+Javascript es difícil. ¿Por qué agregar más reglas?
```
Comment (ramirofarias) [inline www/src/pages/es/why.md:14]:
```suggestion
JavaScript es difícil. ¿Por qué agregar más reglas?
```

File: www/src/pages/es/why.md [hunk 20]
```diff
@@ -0,0 +1,50 @@
...
+
+Creemos firmemente que la experiencia que brinda TypeScript te ayudará a ser un mejor desarrollador. Proporciona retroalimentación en vivo a medida que escribes tu código definiendo los tipos de datos esperados, y proporciona un autocompletado útil en tu editor o te indica con líneas onduladas rojas si estás tratando de acceder a una propiedad que no existe o tratando de pasar un valor del tipo incorrecto, que de otro modo tendrías que depurar más adelante. Ya sea que seas nuevo en el desarrollo web o un profesional experimentado, el "rigor" de TypeScript te brindará una experiencia menos frustrante y más consistente que el estándar JS.
```
Comment (ramirofarias) [inline www/src/pages/es/why.md:16]:
```suggestion
Creemos firmemente que la experiencia que brinda TypeScript te ayudará a ser un mejor desarrollador. Proporciona retroalimentación en vivo a medida que escribes tu código definiendo los tipos de datos esperados, y proporciona un autocompletado útil en tu editor o te grita con líneas onduladas rojas si estás tratando de acceder a una propiedad que no existe o tratando de pasar un valor del tipo incorrecto, que de otro modo tendrías que depurar más adelante. Ya sea que seas nuevo en el desarrollo web o un profesional experimentado, el "rigor" de TypeScript te brindará una experiencia menos frustrante y más consistente que JS estándar.
```

**Review by ramirofarias — CHANGES_REQUESTED** [review]
I added some change suggestions to some lines. Some of them are typing fixes, and the rest are related to the translation to better reflect the normal use in Spanish.

File: www/src/config.ts (hunk 10 above)
Comment (c-ehrlich) [reply www/src/config.ts:169]: Agreed

Comment (Nsttt) [reply www/src/config.ts:169]: 👍

**Comment by andrescv** [comment]
This is ready for review, let me know your comments. I would appreciate a final check from you guys @Nsttt @ramirofarias

File: www/src/config.ts (hunk 10 above)
Comment (Nsttt) [reply www/src/config.ts:344]: This one is still pending ?

**Comment by Nsttt** [comment]
All good from my side. 👍

**Comment by ramirofarias** [comment]
Looks good to me!

**Review by c-ehrlich — APPROVED** [review]
awesome work on this everyone! excited to get it out :)

**Comment by c-ehrlich** [comment]
@amariani @andrescv @hodeinavarro @Nsttt @ramirofarias

thanks for this translation. let me know if you want to be added to `translations.yaml` as codeowners for the spanish docs! it means that you get a notification on github everytime there's a pr for changes to the translation so you can review it.

**Comment by Nsttt** [comment]
> thanks for this translation. let me know if you want to be added to `translations.yaml` as codeowners for the spanish docs!

I'd love to !

**Comment by ramirofarias** [comment]
> @amariani @andrescv @hodeinavarro @Nsttt @ramirofarias
> 
> thanks for this translation. let me know if you want to be added to `translations.yaml` as codeowners for the spanish docs! it means that you get a notification on github everytime there's a pr for changes to the translation so you can review it.

Sure!

**Pull request merged**